code/__HELPERS/priority_announce.dm

```diff
@@ -86,34 +116,104 @@
  * message - the message contents of the announcement.
  * title - the title of the announcement, which is often "who sent it".
  * alert - whether this announcement is an alert, or just a notice. Only changes the sound that is played by default.
- * from - who sent the announcement, in the case of communications consoles or the like.
  * html_encode - if TRUE, we will html encode our title and message before sending it, to prevent player input abuse.
- * players - optional, a list mobs to send the announcement to. If unset, sends to all players.
+ * players - optional, a list mobs to send the announcement to. If unset, sends to all palyers.
  * sound_override - optional, use the passed sound file instead of the default notice sounds.
+ * should_play_sound - Whether the notice sound should be played or not.
+ * color_override - optional, use the passed color instead of the default notice color.
  */
-/proc/minor_announce(message, title = "Attention:", alert, from, html_encode = TRUE, list/players, sound_override)
```

Comment:
Make sure to retain bee's unique captain messages containing the senders name. I see the from is deleted so I will make a comment here.

I tried porting this a few months back and that's a change I had to look out for.

Reply:
am i crazy or is auth_id never fucking passed anywhere anyways

Reply:
Yea you're right, looks like it got removed at some point

Reply:
@Absolucy @Tsar-Salat Bumping for a decision to be made on this change. How important is it, how much do we care and is it something we can just accept as being removed for now considering it seems to have been accidentally removed at some point anyway.